src/abi.cpp

```diff
@@ -163,29 +163,27 @@ struct fill_t {
    abi_type& type;
    int depth;
    template<typename T>
-   auto operator()(T& t) -> result<std::void_t<decltype(resolve(abi_types, t, depth))>> {
-      OUTCOME_TRY(x, resolve(abi_types, t, depth));
+   auto operator()(T& t) -> std::void_t<decltype(resolve(abi_types, t, depth))> {
+      auto x = resolve(abi_types, t, depth);
       type._data = std::move(x);
-      return outcome::success();
    }
    template<typename T>
-   auto operator()(const T& t) -> result<void> {
-      return outcome::success();
+   auto operator()(const T& t) {
    }
 };
 
-result<void> fill(std::map<std::string, abi_type>& abi_types, abi_type& type, int depth) {
+void fill(std::map<std::string, abi_type>& abi_types, abi_type& type, int depth) {
    return std::visit(fill_t{abi_types, type, depth}, type._data);
 }
 
 }
 
 
-result<const abi_type*> eosio::abi::get_type(const std::string& name) {
-   return result<const abi_type*>(::get_type(abi_types, name, 0));
+const abi_type* eosio::abi::get_type(const std::string& name) {
+   return (const abi_type*)(::get_type(abi_types, name, 0));
```

Comment:
How about `static_cast<const abi_type>` instead.

Reply:
No cast is needed at all.

Reply:
Even better.